src/components/KeyboardCompatibleView.js

```diff
@@ -70,6 +70,13 @@ export class KeyboardCompatibleView extends React.PureComponent {
     ) {
       this.setupListeners();
     } else {
+      /**
+       * When leaving an active app state but still mounted we want to reset
+       * animated height values so when active again, animations will properly
+       * adjust the view instead of having no difference in value to animate to
+       */
+      this.setState({ channelHeight: new Animated.Value(0) });
```

Comment:
![IMG_0010](https://user-images.githubusercontent.com/9044436/80148362-f4f28a80-8582-11ea-9c3e-bdd2303fe1e1.png)
this is what happens when I make it `'100%'1

Reply:
@vishalnarkhede also onLayout does get called when your app state transitions back into active

Reply:
@vishalnarkhede technically I can change the initial animated value for channelHeight in the constructor to 0 as well and it should work that means

Reply:
> this is what happens when I make it `'100%'1

It should be string - `this.setState({ channelHeight: new Animated.Value('100%') });`

> technically I can change the initial animated value for channelHeight in the constructor to 0 as well and it should work that means

I will prefer default height to be 100% actually. If something breaks, then better to show full screen message list than 0 height. What do you think?

Reply:
@vishalnarkhede okay I'll figure it out and let you know

Reply:
So we initially set `channelHeight` to `Animated.Value('100%)` and `this.initialHeight = undefined;` . `onLayout` changes `this.initialHeight` to the int height of the view and then sets the state `channelHeight` value to that int value instead of the `'100%'` string. within `keyboardDidShow` we're animating to `finalHeight` which is an int value. then `keyboardDidHide` and `keyboardWillDismiss` have animations to `this.initialHeight` which is an int. So the `Animated.Value` is usually a int (`currentHeight`) to and int (`initialHeight` or `finalHeight`). So I actually think changing the _handleAppStateChange state set that I was doing to `this.initialHeight` instead of `0` makes more sense